cvat/apps/engine/background.py

```diff
@@ -80,24 +84,26 @@ def setup_background_job(self, queue: DjangoRQ, rq_id: str) -> None:
         pass
 
     @abstractmethod
-    def _handle_rq_job_v1(self, rq_job: RQJob, queue: DjangoRQ) -> Optional[Response]:
+    def _handle_rq_job_v1(self, rq_job: Optional[RQJob], queue: DjangoRQ) -> Optional[Response]:
         pass
 
-    def _handle_rq_job_v2(self, rq_job: RQJob, *args, **kwargs) -> Optional[Response]:
-        rq_job_status = rq_job.get_status(refresh=False)
-        if rq_job_status in {
-            RQJobStatus.FINISHED,
-            RQJobStatus.FAILED,
-            RQJobStatus.CANCELED,
-            RQJobStatus.STOPPED,
-        }:
-            rq_job.delete()
+    def _handle_rq_job_v2(self, rq_job: Optional[RQJob], *args, **kwargs) -> Optional[Response]:
+        if not rq_job:
             return None
 
-        return Response(
-            data=f"Export process is already {'started' if rq_job_status == RQJobStatus.STARTED else 'queued'}",
-            status=status.HTTP_409_CONFLICT,
-        )
+        rq_job_status = rq_job.get_status()
+
+        if rq_job_status in {RQJobStatus.STARTED, RQJobStatus.QUEUED}:
+            return Response(
+                data=f"Export process is already {rq_job_status}",
```

Comment:
```suggestion
                data=f"Export process is already started",
```
I think this difference between started and queued is an implementation detail, that should be encapsulated.